webapp/app/js/perf.js

```diff
@@ -110,6 +110,13 @@ perf.controller('PerfCtrl', [ '$state', '$stateParams', '$scope', '$rootScope',
       $scope.$digest();
 
       this.plot.unhighlight();
+
+      //Highlight the points persisted in the url
```

Comment:
Leave a space between '//' and 'Highlight' i.e. //Highlight the points persisted in the url -> // Highlight the points persisted in the url